notebook/build.js

```diff
@@ -0,0 +1,35 @@
+var fs = require('fs');
+var aliasify = require('aliasify');
+var browserify = require('browserify');
+var mkdirp = require("mkdirp");
+var path = require('path');
+
+var aliasifyConfig = {
+    aliases: {
+        jqueryui: 'jquery-ui',
+        termjs: 'term.js',
+        caja: 'google-caja/html-css-sanitizer-minified'
+    },
+    verbose: false
+}
+
+var b = browserify({
+    paths: [
+        __dirname + '/static-src',
+        __dirname + '/static/components',
```

Comment:
I tried using `debowerify` to handle bower components for me, but something about the `..` in the relative paths caused the whole thing to barf.  This is an alternative that allows me to reference bower components, which are not available in npm.